lib/Models/TerriaJsonCatalogFunction.js

```diff
@@ -371,10 +371,17 @@ TerriaJsonCatalogFunction.prototype.invoke = function() {
     if (!defined(parameter.value) || parameter.value === "") {
       return;
     }
-
-    queryParameters[parameter.id] = parameter.formatForService();
+    // determine if parameter is a group of parameters and 
+    // if parameter has groupIsTransparent in its AvailableFormatters
+    // first working prototype and would appreciate feedback.
+    if (parameter.type.endsWith('-group') && ('groupIsTransparent' in parameter.availableFormatters)) {
+        parameter.formatForService().forEach(function(value) { 
+          queryParameters[Object.keys(value)] = value[Object.keys(value)];
```

Comment:
How about making this more explicit?

If each value is of type `{id: string, value: string}`, then you could  rewrite it as `queryParameters[param.id] = param.value;`

Reply:
the values were not necessarily like that, one example I had was 
value = {"C": "true"} which would become 
queryParameters["C"] = true.
can switch to that.